src/locale/index.js

```diff
@@ -0,0 +1,11 @@
+import en from "./en.json";
```

Comment:
Follow up: fixed using a dynamic import in 1434730 
let me know if this works for you?

Reply:
Thanks for fixing this :)

Reply:
No problem, not sure how much of an actual fix it is as the locales will always be imported? Later I was thinking that maybe if we can detect the users language and then if it’s not English we do all the translation stuff, but this might make things too complex.